### PR TITLE
Calculator components PR

### PR DESCRIPTION
**Pull request by kkasaija**
### Milestone Reached
- [x] Set up `React` App
- [x] Creatd a directory called components.
- [x] Inside components, created a new Calculator.js file.
- [x] In Calculator.js, created a React component that matches this design:
calculator

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,12 @@
...
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1>Basic Calculator</h1>
```
Comment (1GORDON) [inline src/App.js]: Great job @Kasaija-Kenneth 🤝

- [ ] Kindly make sure for this project all components are class-based. You may change this App function component into a class-based component.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Kasaija-Kenneth ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (kkasaija) [reply src/App.js]: Noted, with thanks

**Review by BenMukebo — APPROVED** [review]
### Hi @Kasaija-Kenneth, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Gitflow correctly used 🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [OPTIONAL] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

00Mit


- The reason for this is that there is no `MIT.md` file in the root directory of this project. Let's update this section(`License`) with a link to the license page, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkasaija** [comment]
Thank you for that reminder. I am going to work on the license.

**Pull request merged**